test/fixtures/refunded-order-partial.json

```diff
@@ -0,0 +1,30 @@
+{
+  "input": {
+    "userId": "user-id",
+    "type": "track",
+    "event": "Order Refunded",
+    "properties": {
+      "orderId": 12345,
+      "products": [
```

Comment:
`products` doesn't seem to be spec'd https://segment.com/docs/spec/ecommerce/v2/#order-refunded?